utils/install-common.sh

```diff
@@ -56,8 +56,7 @@ install_openresty_deb() {
 install_openresty_rpm() {
     # install openresty and openssl111
     yum-config-manager --add-repo https://openresty.org/package/centos/openresty.repo
-    yum install -y pcre-devel
-    yum install -y openresty openresty-openssl111-devel
+    yum install -y openresty openresty-openssl111-devel pcre-devel
```

Comment:
We should also add pcre itself as the dependency for the fpm package. It can be done in the next PR.